cmd/cmd.go

```diff
@@ -953,5 +952,21 @@ Environment Variables:
 		deleteCmd,
 	)
 
+	rootCmd.SetUsageTemplate(rootCmd.UsageTemplate() + `
```

Comment:
this adds quite a bit of weight to the bottom of the default help text for `ollama`

```
go run . 
Usage:
  ollama [flags]
  ollama [command]

Available Commands:
  serve       Start ollama
  create      Create a model from a Modelfile
  show        Show information for a model
  run         Run a model
  pull        Pull a model from a registry
  push        Push a model to a registry
  list        List models
  cp          Copy a model
  rm          Remove a model
  help        Help about any command

Flags:
  -h, --help      help for ollama
  -v, --version   Show version information

Use "ollama [command] --help" for more information about a command.

Environment Variables:

    OLLAMA_HOST
        The host:port or base URL of the ollama server
	(default "http://localhost:11434")

	This is used by all commands that interact with the ollama server.
        (e.g. run, pull, push, list, etc.)

        Examples:
            "127.0.0.1:11434"
            "http://localhost:11434"
            "https://api.example.com"
```

is there a way we could put it under individual subcommands (except for `serve`)?

Reply:
We could add more weight above which is probably a good thing, or move envs to their own help topic if this isn't feeling right atm.

Reply:
What might help:
- Add `OLLAMA_HOST` to the relevant subcommands, which would also remove `This is used by all commands that....` as help text. It's also a bit funky that `Use "ollama [command] --help" for more information about a command.` is before the env var definitions
- Keep each env var to one line, if it's at all possible (similar to flags)

```
ollama run -h
Run a model

Usage:
  ollama run MODEL [PROMPT] [flags]

Flags:
      --format string   Response format (e.g. json)
  -h, --help            help for run
      --insecure        Use an insecure registry
      --nowordwrap      Don't wrap words to the next line automatically
      --verbose         Show timings for response

Environment Variables:
  OLLAMA_HOST        The host:port or base URL of the Ollama server (e.g. http://localhost:11434)
```

Reply:
Otherwise this is much needed!

Reply:
The language was a way to avoid the need to keep up with all docs for any new sub commands that may (even inadvertently) be affected by the env var. 

Reducing docs to one line was a pass I made initially but took it a little further to provide examples which seemed helpful since the env is more than just a url or basic host:port.

Maybe we link to a wiki page with more details and use the cli help as hints instead?

Happy to trim it all down too, land this, and take another pass later in another PR.